index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse Project</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<h1>Hello Microverse!</h1>
```

Comment:
Please be sure to include this line inside the body tag. This will improve our semantics and give our code organization a bit more sense because the `body` tag [_contains all the contents of an HTML document, such as headings, paragraphs, images, hyperlinks, tables, lists, etc._](https://www.w3schools.com/tags/tag_body.asp#:~:text=Definition%20and%20Usage,element%20in%20an%20HTML%20document.)